unicode-data/unicode-data.cabal

```diff
@@ -121,10 +120,13 @@ executable ucd2haskell
     buildable: True
     build-depends:
         base             >= 4.7   && < 4.17
-      , streamly         >= 0.8   && < 0.9
-      , getopt-generics  >= 0.13  && < 0.14
+      , binary           >= 0.8.8 && < 0.9
+      , bytestring       >= 0.10  && < 0.12
       , containers       >= 0.5   && < 0.7
       , directory        >= 1.3.6 && < 1.3.7
+      , getopt-generics  >= 0.13  && < 0.14
+      , streamly         >= 0.8   && < 0.9
+      , text             >= 1.1.1 && <=1.2.5.0 || >=2.0 && <2.1
```

Comment:
`binary`, `bytestring` and `text` dependencies are redundant.

Reply:
Done